packages/web3-net/src/rpc_method_wrappers.ts

```diff
@@ -0,0 +1,13 @@
+import { Web3Context } from 'web3-core';
+
+import * as rpcMethods from './rpc_methods';
+import { NetRPCApi } from './types';
+
+export const getId = async (web3Context: Web3Context<NetRPCApi>) =>
+	rpcMethods.getId(web3Context.requestManager);
+
+export const isListening = async (web3Context: Web3Context<NetRPCApi>) =>
```

Comment:
This method doesn't need to exist as an `rpc_method_wrapper`. The idea behind the wrappers is for functions that require some sort of formatting or parameters that have default values. I think you can remove `isListening` from here and import it directly from `rpc_methods.ts` into `packages/web3-net/src/index.ts`

Reply:
it is implemented like in `web3-eth` https://github.com/ChainSafe/web3.js/blob/ea41c556130558b37305767a39f25b4306dc0e50/packages/web3-eth/src/rpc_method_wrappers.ts#L31
assume it is because easier testing